lib/tagging_sync_verifier.rb

```diff
@@ -31,22 +32,24 @@ def verify_taggings_are_in_sync
       from_content_store
     )
 
-    if comparison.same?
-      puts "Taggings are in sync!"
+    only_in_content_api = comparison.diffs[:only_a]
+    only_in_content_store = comparison.diffs[:only_b].select { |content_id, _| from_content_api[content_id] }
```

Comment:
I had trouble interpreting the logic here, and I had to go look at the Comparer to work out what it's doing. My understanding is that `only_b` will return any key, value pairs that are in B but not A, and this line will filter all of them where the key (content id) is not in A.

I think changing this to `select {from_content_api.include?(content_id)}` would help readability here, and maybe some of the variables could be renamed.